Source/SplineFollow/Public/Components/SFSplineFollowingInterface.h

```diff
@@ -9,6 +9,7 @@ class USplineComponent;
 struct FSFFollowSplineInfos;
 
 DECLARE_DELEGATE_OneParam( FSWOnSplineFollowingReachedPositionDelegate, float );
+DECLARE_DYNAMIC_MULTICAST_DELEGATE_OneParam( FSWOnSplineFollowingToggleMovementChangedDelegate, bool, enabled );
```

Comment:
the delegate type should be `FSFOnSplineFollowingToggleMovementChangedDelegate`

(same for the delegate above)